examples/basic/index.html

```diff
@@ -153,15 +151,23 @@
 }
 
 function onResize() {
-  console.log('Resizing to %s x %s.', window.innerWidth, window.innerHeight);
-  effect.setSize(window.innerWidth, window.innerHeight);
-  camera.aspect = window.innerWidth / window.innerHeight;
-  camera.updateProjectionMatrix();
+  // The delay ensures the browser has a chance to layout
+  // the page and update the clientWidth/clientHeight.
+  if(!this.resizeDelay) {
+    this.resizeDelay = setTimeout(() => {
```

Comment:
nit: for the sake of browser compat (i.e., iOS 9 and IE 11), can you use `function () { … }`?

not a huge deal, but it's the only arrow function here.

you should be able to just replace `this.resizeDelay` with `resize.resizeDisplay`, and it should continue to work the same.